Shared/Samples/Add raster from file/AddRasterFromFileView.swift

```diff
@@ -16,23 +16,27 @@ import ArcGIS
 import SwiftUI
 
 struct AddRasterFromFileView: View {
-    /// Makes a map with a raster layer.
-    private static func makeMap() -> Map {
-        /// A map with a standard imagery basemap style.
-        let map = Map(basemapStyle: .arcGISImageryStandard)
-        // Gets the Shasta.tif file URL.
-        let shastaURL = Bundle.main.url(forResource: "Shasta", withExtension: "tif", subdirectory: "raster-file/raster-file")!
-        // Creates a raster with the file URL.
-        let raster = Raster(fileURL: shastaURL)
-        // Creates a raster layer using the raster object.
-        let rasterLayer = RasterLayer(raster: raster)
-        // Adds the raster layer to the map's operational layer.
-        map.addOperationalLayer(rasterLayer)
-        return map
+    private class Model: ObservableObject {
+        /// A map with imagery basemap and a raster layer.
+        let map: Map
+        
+        init() {
+            /// A map with a standard imagery basemap style.
+            let map = Map(basemapStyle: .arcGISImageryStandard)
+            // Gets the Shasta.tif file URL.
+            let shastaURL = Bundle.main.url(forResource: "Shasta", withExtension: "tif", subdirectory: "raster-file/raster-file")!
```

Comment:
Instead of [setting](https://docs.swift.org/swift-book/LanguageGuide/Initialization.html#ID232) the default property value with a closure (which is effectively similar to a static func), I moved the raster into the initializer. It might solve the problem in #48 .

Reply:
I got a crash here opening the sample for the first time.

Reply:
🤦  My speculation is wrong again.